R/GroupSequentialDesign.R

```diff
@@ -61,21 +61,22 @@ setMethod("n2", signature("GroupSequentialDesign", "numeric"),
           }
 )
 
-#' @template d
+#' @param n1 \code{GroupSequentialDesign} object to convert
```

Comment:
we had a generic TwoStageDesign(...) previously - that's nonsense, it cannot handle method dispatch properly. it's TwoStageDesign(n1, ...) now since this is the most common argument name